### DISCUSSION TITLE
Convention / notation for declaring a page's docs type

### DISCUSSION THREAD

**Discussion by bennuttall** [evildmp/diataxis-documentation-framework#26] [Ideas]
I wondered if anyone had any suggestions for a convention or notation to declare what "type" a docs page is (tutorial, how-to, reference, explanation) - perhaps a comment at the top of the file:

```
.. docs-type: reference
```

maybe:

```
.. docs-type: reference https://diataxis.fr/reference/
```

or just:

```
.. https://diataxis.fr/reference/
```

The intention would be to hint to editors which type of documentation the page is aiming to be, so that they don't inadvertently start adding explanation to a reference page. Or it could inform/remind people of the documentation system itself.

Maybe it doesn't need to be an agreed convention, just something someone decides to do of their own accord (maybe I will).

What do you think?

**Comment by evildmp** [comment]
I think this is a really good idea, especially of using a URL.

Also while:

````
..  https://diataxis.fr/reference/
````

is probably OK, generally it's better to put rST comments on a separate line, in case something decides to interpret the first word it finds as a directive - so:

````
..  
    docs-type: reference https://diataxis.fr/reference/
````
